src/components/drawer/Swipeable.tsx

```diff
@@ -123,7 +125,8 @@ export default class Swipeable extends Component<Props, StateType> {
       if (!this.dragThresholdReached && x >= threshold && x < threshold + 10) {
         // move item right
         this.dragThresholdReached = true;
-        onToggleSwipeLeft({rowWidth, leftWidth, dragX: x, triggerHaptic: true});
+        HapticService.triggerHaptic(HAPTIC_METHOD, 'Drawer');
+        onToggleSwipeLeft({rowWidth, leftWidth, dragX: x});
```

Comment:
I think we should keep the haptic optional, not all users will want to use it.

Reply:
Why not get the haptic method from the HapticService, i.e. `HapticMethods.impactMedium`?

Reply:
I'll be able to do that only when I'll fix the `HapticMethods` in the `HapticService` as we talked about in our dev sync.
I'm waiting for this PR to be merged to do so

Reply:
Ok. Add the other PR as a dependency and we'll merge this after it.

Reply:
Is the dependency merged?

Reply:
Yes :)